### ISSUE TITLE
Non-breaking error in Update Password page

### ISSUE THREAD

**Issue by shitwolfymakes** [automatic-ripping-machine/automatic-ripping-machine#656]
- [x] I have checked the wiki
- [x] I have searched for similar issues

**Describe the bug**
`update_password` page errors when a non-existant username is entered into the change password form

**To Reproduce**
1. Log in to ARMui
2. Select "Change admin password"
3. Try to change the password for a user that doesn't exist

### Environment
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.5 LTS"
```
ARM Version: 2.6.9

**Comment by microtechno9000** [comment]
See the issue, it is assuming that there is a user entered and not pulling any new info from the database
I have fixed with commit [335ef8e](https://github.com/automatic-ripping-machine/automatic-ripping-machine/pull/650/commits/335ef8edbdb19baf325f29e1da3540c66aad0567)
- Since ARM only operates with one user 'admin', it will pull the first row from the database and load to the form. The username field is then readonly to prevent any changes to this field

**Comment by shitwolfymakes** [comment]
Closed by #650
